public/assets/ts/exam-bank.ts

```diff
@@ -110,10 +110,12 @@ class ExamBankFrontend {
       newRow.id = "";
       newRow.innerHTML = newRow.innerHTML
         .replace("$COURSE-NAME$", `${exam.department} ${exam.courseCode}`)
+        .replace("$COURSE-DEPT$", exam.department)
+        .replace("$COURSE-CODE$", exam.courseCode)
         .replace("$OFFERING$", exam.termName)
         .replace("$TYPE$", exam.type)
-        .replace("$EXAM-FILE$", `/exams/${exam.examFile}`)
-        .replace("$SOLUTION-FILE$", `/exams/${exam.solutionFile}`)
+        .replace(/EXAM-FILE/g, `/exams/${exam.examFile}`)
```

Comment:
sorry for not catching this - doesn't this require you to make a change in the exam-bank-table component?

Reply:
oops this is an artifact of working on linking up the frontend earlier, i'll remove it for now!